hat/assets/js/apps/Iaso/components/filters/FiltersComponent.js

```diff
@@ -125,7 +125,11 @@ class FiltersComponent extends React.Component {
                                                 filter.callback,
                                             )
                                         }
-                                        value={filterValue}
+                                        value={
+                                            filter.loading
+                                                ? undefined
+                                                : filterValue
+                                        }
```

Comment:
Not blocking but it would be more consistent to add an `if` statement line 90, like
```javascript
if(filter.loading){
  filterValue=undefined  
}
```

 I think it will save a minute in a later refactor

Reply:
The refactor that I'll do if I remember correctly 😁